ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -156,4 +156,39 @@ QtObject {
     function resolveENS(value) {
         mainModuleInst.resolveENS(value, "")
     }
+
+    property var networkConnectionModuleInst: networkConnectionModule
+
+    function getNetworkNameByChainId(chainId) {
+        for(var i = 0; i < allNetworks.count; i++) {
+            let networkChainId = allNetworks.rowData(i, "chainId") * 1
+            if(networkChainId === chainId) {
+                return allNetworks.rowData(i, "chainName")
+            }
+         }
+        return ""
+    }
+
+    function getChainIdsJointString(chainIdsDown) {
+        let jointChainIdString = ""
+        for (const chain of chainIdsDown) {
+            jointChainIdString = (!!jointChainIdString) ? jointChainIdString + " & " : jointChainIdString
```

Comment:
Uhm... :police_officer: Normally you'd call https://doc.qt.io/qt-5/qlocale.html#createSeparatedList but this is not exposed to QML :/

Reply:
Damn, once we move to c++ then!!